gradle-palantir-java-format/src/main/java/com/palantir/javaformat/gradle/PalantirJavaFormatSpotlessPlugin.java

```diff
@@ -28,10 +29,14 @@ public class PalantirJavaFormatSpotlessPlugin implements Plugin<Project> {
     public void apply(Project project) {
         project.getRootProject().getPluginManager().apply(PalantirJavaFormatProviderPlugin.class);
 
+        SpotlessInterop spotlessInterop =
+                project.getRootProject().getObjects().newInstance(SpotlessInterop.class, project.getRootProject());
```

Comment:
nit: let's extract `project.getRootProject()` to a variable? 

Also, maybe `project.getObjects()` does the same thing.